plugins/BEdita/Core/src/Model/Action/DeleteEntityAction.php

```diff
@@ -41,6 +43,11 @@ protected function initialize(array $data)
      */
     public function execute(array $data = [])
     {
-        return $this->Table->delete($data['entity']);
+        $success = $this->Table->delete($data['entity']);
```

Comment:
Would `$this->Table->deleteOrFail($data['entity'])` do the job anyways?

It throws a `PersistenceFailedException` that provides some further details in logs, and still results in a **500 Internal Server Error** response, so IMHO it should be preferred.